src/dirs.proj

```diff
@@ -17,6 +17,9 @@
     <Project Include="XMakeTasks\UnitTests\Microsoft.Build.Tasks.UnitTests.csproj" />
     <Project Include="XMakeBuildEngine\UnitTests\Microsoft.Build.Engine.UnitTests.csproj" />
     <Project Include="XMakeBuildEngine\UnitTestsPublicOM\Microsoft.Build.Engine.OM.UnitTests.csproj" />
+
+    <Project Include="Test\Projects\Dependency\Dependency.csproj" />
+    <Project Include="Test\Projects\TaskWithDependency\TaskWithDependency.csproj" />
```

Comment:
The sample projects should go into this other dirs.proj: [https://github.com/Microsoft/msbuild/blob/xplat/Samples/dirs.proj](https://github.com/Microsoft/msbuild/blob/xplat/Samples/dirs.proj)